Latest build 5.45.0 gives gui declaration file error

The latest build 5.45.0 give typescript error with Babylon guy declaration file:

```
Assets/[Babylon]/Template/Typings/babylon.gui.d.ts(594,96): error TS2792: Cannot find module 'core/Engines/ICanvas'. Did you mean to set the 'moduleResolution' option to 'node', or to add aliases to the 'paths' option?
```

--- a/packages/dev/gui/src/2D/controls/container.ts
+++ b/packages/dev/gui/src/2D/controls/container.ts
@@ -7,7 +7,7 @@ import type { AdvancedDynamicTexture } from "../advancedDynamicTexture";
 import { RegisterClass } from "core/Misc/typeStore";
 import type { PointerInfoBase } from "core/Events/pointerEvents";
 import { serialize } from "core/Misc/decorators";
-import type { ICanvasGradient, ICanvasRenderingContext } from "core/Engines/ICanvas";
+import type { ICanvasRenderingContext } from "core/Engines/ICanvas";
 import { DynamicTexture } from "core/Materials/Textures/dynamicTexture";
 import { Texture } from "core/Materials/Textures/texture";
 import { Constants } from "core/Engines/constants";
@@ -345,7 +345,7 @@ export class Container extends Control {
         }
     }
 
-    protected _getBackgroundColor(context: ICanvasRenderingContext): string | ICanvasGradient {
+    protected _getBackgroundColor(context: ICanvasRenderingContext) {
         return this._backgroundGradient ? this._backgroundGradient.getCanvasGradient(context) : this._background;
     }
 

--- a/packages/dev/gui/src/2D/controls/rectangle.ts
+++ b/packages/dev/gui/src/2D/controls/rectangle.ts
@@ -2,7 +2,7 @@ import { Container } from "./container";
 import type { Measure } from "../measure";
 import { RegisterClass } from "core/Misc/typeStore";
 import { serialize } from "core/Misc/decorators";
-import type { ICanvasGradient, ICanvasRenderingContext } from "core/Engines/ICanvas";
+import type { ICanvasRenderingContext } from "core/Engines/ICanvas";
 
 /** Class used to create rectangle container */
 export class Rectangle extends Container {

--- a/packages/dev/gui/src/2D/controls/rectangle.ts
+++ b/packages/dev/gui/src/2D/controls/rectangle.ts
@@ -73,7 +73,7 @@ export class Rectangle extends Container {
         return 0;
     }
 
-    protected _getRectangleFill(context: ICanvasRenderingContext): string | ICanvasGradient {
+    protected _getRectangleFill(context: ICanvasRenderingContext) {
         return this._getBackgroundColor(context);
     }
 

